server/dex/dex.go

```diff
@@ -129,8 +168,19 @@ func (dm *DEX) Stop() {
 	}
 }
 
-// TODO
-//func (dm *DEX) handleDEXConfig (conn comms.Link, msg *msgjson.Message) *msgjson.Error { }
+func (dm *DEX) handleDEXConfig(conn comms.Link, msg *msgjson.Message) *msgjson.Error {
```

Comment:
No test coverage, but this is pretty trivial.  What happens if `msg` has marshaling errors all depends on what the `Link` does, and that is tested in the `comms` package.

Reply:
Agree. We can hit this through the front end with some live testing eventually.